tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -988,6 +1014,82 @@ public function getValue()
                 }
                 EOD,
         ];
+
+        yield 'import even if partly importable using namespace' => [
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+                use Ns\A\B;
+                use Ns\A\B\C;
+
+                new A();
+                new B();
+                new C();
+                EOD,
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+
+                new \Ns\A();
+                new \Ns\A\B();
+                new \Ns\A\B\C();
+                EOD,
+            ['import_symbols' => true],
+        ];
+
+        yield 'do not import relative symbols if not configured so - namespaced' => [
+            <<<'EOD'
+                <?php
+
+                namespace Ns;
+
+                new A();
+                new A\B();
+                new A\B\C();
+                EOD,
+            null,
+            ['import_symbols' => true],
+        ];
+
+        yield 'do not import relative symbols if not configured so - global with use' => [
+            <<<'EOD'
+                <?php
+
+                use A;
+                use B\B2\C2;
+
+                new A();
+                new A\B();
+                new A\B\C();
+                new C2();
```

Comment:
I don't understand why it's not like this:

```suggestion
                use A;
                use A\B;
                use A\B\C;
                use B\B2\C2;

                new A();
                new B();
                new C();
                new C2();
```

Since this is global namespace, all of these symbols can be treated as FQCNs with implicit leading backslash. So why only `B\B2\C2` is imported?

Reply:
`A\B` is "relative type" because of `A` import.